environment.yml

```diff
@@ -64,7 +64,7 @@ dependencies:
     # SEE PIP DEPENDENCIES FOR MORE
 
   # typechecking
-  - mypy ~=0.910
+  - mypy ==0.990
```

Comment:
https://nsidc.org/jira/browse/QGR-145

Reply:
Context from ticket: 0.991 finds new errors that <=0.990 doesn't